kuma/static/img/web-docs-sprite.svg

```diff
@@ -0,0 +1,9 @@
+</svg>
```

Comment:
Missing `eol` also, worth running this through SVGOMG. We should probably make SVGO part of the build process as we have more and more SVG assets.